src/ini_to_core_reader.cpp

```diff
@@ -82,6 +92,14 @@ void INIToCoreReader::parse( const string& filename ) throw ( h_exception ) {
 int INIToCoreReader::valueHandler( void* user, const char* section, const char* name,
                                   const char* value )
 {
+    #ifdef USE_RCPP
+    // Load R functions for path management
+    Rcpp::Environment base("package:base");
+    Rcpp::Function normalizePath = base["normalizePath"];
+    Rcpp::Function dirname = base["dirname"];
+    Rcpp::Function filepath = base["file.path"];
+    #endif
```

Comment:
If we did want to use `R.utils::isAbsolutePath`, we would have to add the following here:

```cpp
Rcpp::Environment rutils("package::R.utils");
Rcpp::Function isAbsolutePath = rutils["isAbsolutePath"];
```

Then, further down, I _think_ we could just do something like this:

```cpp
if (isAbsolutePath(csvFilePath)) {
   ...
}
```

Reply:
Given your comments about this, I'd say let's just not worry about it for now.  Your current solution is likely sufficient and if I'm wrong we can deal with it then.